Fix #32 - MacOS crash when Desktop becomes the active window

uglow opened the pull request:
Fixes #32. 

The Desktop window does not appear in the `window` collection because of the `excludeDesktopElements` flag - which is reasonable as you probably don't want icons and the System menu showing up in the results. However, this causes the crash that was identified in issue #32.

This PR prevents the system from crashing out when the active window is not in the `window` collection by return a "fake" Desktop result.

uglow commented:
@sindresorhus Good to merge?

matthewjumpsoffbuildings commented:
im having this issue too, hoping this fixes it

uglow commented:
@tiangolo @sindresorhus - are you able to review and merge, please?

sindresorhus commented on Sources/active-win/main.swift:
```diff
@@ -54,3 +54,19 @@ for window in windows {
+	  "bundleId": app.bundleIdentifier!,
+	  "path": app.bundleURL!.path
+  ]
+]
```
If we go with this solution, you have to fill out all the fields.

sindresorhus commented:
I think a better solution would be to admit that there can't always be an active window, and return null in such scenarios.

uglow commented:
If we return null it would be a (more) breaking change than this PR. But I don’t mind.

sindresorhus commented:
I'm ok with that.

matthewjumpsoffbuildings commented:
i also think returning null is fine, its easy enough to modify code to check if null

uglow commented:
Updated to return null. Works locally when calling `sleep 5 && ./main`. Added test to demonstrate that `JSON.parse('null')` returns `null` (ie. retuning 'null' doesn't break the JSON parser).

uglow commented:
@sindresorhus Good to merge?

uglow commented:
Btw, I've just created a tool called `active-win-log` ([link](https://www.npmjs.com/package/active-win-log)) which uses _this package_ to create a usage log. I'm using it to help me keep track of time spent on projects when working remotely. Thanks for writing this package! 👍🏻

sindresorhus commented:
@uglow Cool use-case. You should do a PR to add it to the readm here. Under a new "Users" section or something ;)
